src/Connection_Counter.php

```diff
@@ -4,6 +4,8 @@
 
 namespace atk4\dsql;
 
+@trigger_error('Use atk4\dsql\Debug\Profiler\Connection instead', E_USER_DEPRECATED);
```

Comment:
You should always review CS fixes manually! I tested it now and with @ is does not emit any message!

If needed, write it as `'trigger_error'(...` (yes, with single quotes)

Reply:
It does not work without @ either as warning is triggered immediately

Reply:
https://3v4l.org/QepiX

Reply:
This is clear. The problem is that the files are autoloaded by phpunit coverage so the warning is triggered during testing always

Reply:
Does my "test" commit solves that?